### DISCUSSION TITLE
Deployed project page not working

### DISCUSSION THREAD

**Discussion by buseugrs** [vercel/community#6289] [Help] [answered]
### Summary

I have deployed page on Vercel. It was working correctly but for now, when I try to open page there is an error like "This site cannot be reached".

### Example

https://multi-select-autocomplete-seven.vercel.app/

### Steps to Reproduce

![Screenshot 2024-03-27 162118](https://github.com/vercel/community/assets/112654875/dbdd2a25-34a3-48c8-9b38-666ac5d57c34)

**Comment by OmerFarukOruc** [comment]
Apparently Turkish govt blocked Vercel for a reason. You can try using VPN to bypass :))

**Reply by buseugrs** [reply]
Bu yeni gelişen bir durum mu acaba ? Birkaç gün öncesine kadar böyle problem yoktu çünkü, şaşkınım..

**Reply by OmerFarukOruc** [reply]
bende yeni fark ettim, ne olduğunu anlamak için buraya geldim sonra ekşiye bakayım dedim oraya da biri [entry](https://eksisozluk.com/entry/162868146) girmiş. anlaşılan tr den erişime engellemişler.

**Reply by serter1** [reply]
Bıktık ya gerçekten

**Reply by CllDMR-ekonobi** [reply]
Bu saçmalık ya

**Reply by BirthMarkApp** [reply]
Vasatlıklar cumhuriyeti

**Comment by amyegan** [comment, marked as answer]
Hey everyone. Thanks for reporting this issue. It looks like Vercel sites are being blocked by an ISP in your region. You may be able to access the sites by using alternative DNS servers, such as Google DNS or Cloudflare DNS.

Our team has reached out for help getting these sites unblocked, but you should also contact your internet service provider and request that they not block access to `*.vercel.app`.
